doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -20,7 +20,7 @@ The following is a Requirements Traceability Matrix (RTM) mapping Test Case, and
 
 ### Test Case Traceability to RA2 Requirements
 
-This section focuses on the test cases covering the requirements in [RA-2 Chapter 4.9](../../../ref_arch/kubernetes/chapters/chapter04.md#49-kubernetes-workloads) for Kubernetes workloads.
+This section focuses on the test cases covering the requirements in [RA-2 Chapter 4.9](../../../ref_arch/kubernetes/chapters/chapter04.md) for Kubernetes workloads.
```

Comment:
The reason for the failing link check seems to be that Sphinx rightfully does not use the header numbering when generating the header reference. In the final build on read the docs, this is the resulting link:

https://cntt.readthedocs.io/en/latest/ref_arch/kubernetes/chapters/chapter04.html#kubernetes-workloads

So, instead of removing the reference to the heading, I suggest to change it like this: 

```suggestion
This section focuses on the test cases covering the requirements in [RA-2 Chapter 4.9](../../../ref_arch/kubernetes/chapters/chapter04.md#kubernetes-workloads) for Kubernetes workloads.
```

Reply:
At the end of the day we will leverage rst link (internal or intersphinx)

Reply:
done